server.js

```diff
@@ -7949,7 +7949,7 @@ cache(function (data, match, sendBadge, request) {
 }));
 
 // Any badge.
-camp.route(/^\/(:|badge\/)(([^-]|--)*?)-(([^-]|--)*)-(([^-]|--)+)\.(svg|png|gif|jpg)$/,
+camp.route(/^\/(:|badge\/)(([^-]|--)*?)-?(([^-]|--)*)-(([^-]|--)+)\.(svg|png|gif|jpg)$/,
```

Comment:
With this regex, if the dash is excluded, does the "message" capture group get everything, while the "label" gets an empty string?

Reply:
Sure does, Handles the same as if you currently used `https://img.shields.io/badge/-message-blue.svg`
![](https://img.shields.io/badge/-message-blue.svg)
But without the need for the `-`: `http://img.shields.io/badge/message-blue.svg`